JitsiMeetJS.js

```diff
@@ -385,6 +385,7 @@ var LibJitsiMeet = {
     util: {
         ScriptUtil: ScriptUtil,
         RTCUIHelper: RTCUIHelper,
+        RTCBrowserType: RTCBrowserType,
```

Comment:
Exposing RTCBrowserType from the lib is a long running internal discussion and we do not want to expose it for now. So can you use a workaround in the jitsi-meet code, where this thing is used and do not expose RTCBrowserType?
There is similar thing here https://github.com/jitsi/jitsi-meet/blob/8745efb81f7aa4fb1790610a47b083722a871645/modules/UI/videolayout/SmallVideo.js#L648-L648
It is not a clean solution, but expanding the API of the lib and then removing stuff from the API can be more painful than fixing those hacks.